src/compiler/config/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const DEFAULT_DEV_MODE = false;
+export const DEFAULT_HASHED_FILENAME_LENTH = 8;
+export const MIN_HASHED_FILENAME_LENTH = 4;
+export const MAX_HASHED_FILENAME_LENTH = 32;
```

Comment:
Can we fix the spelling while we're at it? 😂 
```suggestion
export const MIN_HASHED_FILENAME_LENGTH = 4;
export const MAX_HASHED_FILENAME_LENGTH = 32;
```

Reply:
`LENTH` haha